packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -5,11 +5,141 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../globals/imports';
-
 @mixin leadspace {
   .#{$prefix}--leadspace {
-    background-color: transparent;
+    position: relative;
+    padding-top: carbon--rem(370px) / carbon--mini-units(40) * 100%;
+    height: 0;
+    background: $ui-background;
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__container {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    &__overlay {
+      @include carbon--make-container;
+
+      padding-top: $layout-03;
+      padding-bottom: $layout-03;
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+      max-width: none;
+      background-image: linear-gradient(
+        to bottom,
+        rgba($ui-background, 1),
+        transparent
+      );
+    }
+
+    &__content {
+      position: absolute;
+      bottom: $layout-03;
+    }
+
+    .#{$prefix}--leadspace__title {
+      @include carbon--type-style(display-01, true);
+
+      color: $text-01;
+    }
+
+    .#{$prefix}--leadspace__desc {
+      @include carbon--type-style(expressive-heading-03, true);
+
+      color: $text-01;
+    }
+
+    &__ctas {
+      display: flex;
+      flex-direction: column-reverse;
+      margin-top: $layout-01;
+    }
+
+    .#{$prefix}--btn {
+      width: 100%;
+      margin-top: $layout-01;
+    }
+  }
+
+  .#{$prefix}--leadspace--productive .#{$prefix}--leadspace__title {
+    @include carbon--type-style(expressive-heading-06, true);
+  }
+
+  @include carbon--breakpoint(md) {
+    .#{$prefix}--leadspace {
+      padding-top: carbon--mini-units(50) / carbon--mini-units(84) * 100%;
+
+      &__overlay {
+        background-image: linear-gradient(
+          to right,
+          rgba($ui-background, 1),
+          transparent 50%
+        );
+      }
+
+      &__content {
+        width: 100%;
+      }
+
+      .#{$prefix}--leadspace__title {
+        @include carbon--make-row;
+        @include carbon--make-col-ready;
+        @include carbon--make-col(7, 8);
+      }
+
+      .#{$prefix}--leadspace__desc {
+        width: 50%;
+      }
+
+      &__ctas {
+        flex-direction: row-reverse;
```

Comment:
Should the CTA be apart of the same column as the desc? It appears to just be floating outside of the grid.

![image](https://user-images.githubusercontent.com/3793636/65165573-aab19580-da04-11e9-8edb-f8ff648766e4.png)

Reply:
I chose not to put the CTAs in the same column as the desc as the desc can have a wider or smaller width than the cta div. Do I need to wrap a row around the CTAs? It's always going to be positioned bottom left aligned.